README.md

```diff
@@ -15,10 +42,11 @@ $npm install --save narwin-pack
 - [postcss-svgo](https://github.com/ben-eb/postcss-svgo),
 - [autoprefixer](https://github.com/postcss/autoprefixer),
 - [postcss-hexrgba](https://github.com/seaneking/postcss-hexrgba)
+- [postcss-discard-comments](https://github.com/ben-eb/postcss-discard-comments)
 
 ## Legal
 
-[DockYard](http://dockyard.com), Inc &copy; 2015
+[DockYard](http://dockyard.com), Inc &copy; 2017
```

Comment:
Should we do `2015-2017`? I recall reading a while back that on updated content, including the beginning year & range provides additional defense in case of any copyright challenge.

Reply:
We currently only do single copyright years in other repos, and for DockYard.com. We usually update the year with a major version bump.